e2e/app/page/workspace-base.ts

```diff
@@ -57,32 +58,32 @@ export default abstract class WorkspaceBase extends AuthenticatedPage {
    * Click Datasets subtab in Data page.
    * @param opts
    */
-  async openDatasetsSubtab(opts: {waitPageChange?: boolean} = {}): Promise<void> {
-    return this.openTab(TabLabels.Datasets, opts);
+  async openDatasetsSubtab(): Promise<void> {
+    return this.openDataPage().then(() => this.openTab(TabLabels.Datasets, {waitPageChange: false}));
```

Comment:
Could you also use `await` here?

Reply:
`return` makes `await` redundant. same outcome with or without it.